docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,345 @@
+# Pose Detection
+
+Pose Detection is a new feature of Media Vision Inference API since Tizen 6.0. This feature provides landmark detection. Besides, it defines landmarks and parts of a human body to help detect a human pose with Motion Capture (MoCap) file, which you can create or edit using various tools.
+
+## Background
+
+In Tizen, human body pose landmarks and body parts are defined as follows:
+
+**Figure: Definition of human body pose landmarks and body parts**
+
+![Body pose](./media/mediavision_pose_tizen_def.png)
+
+The Pose landmark detection models are available in Open Model Zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or public GitHub site such as [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). The public pose models provide landmark information, such as the number of landmarks and locations. To use them correctly, you must map the information to landmarks based on the definition. For example, you can use the [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile), which provides 14 landmarks as follows:
+
+![Body pose](./media/mediavision_pose_public_model_def.png),
+
+In this model, `-1` denotes that there are no landmarks. Using this landmark information, you can create a mapping file. Suppose you create a mapping file with the name `pose_mapping.txt,` then you can populate the `pose_mapping.txt` file as follows:
```

Comment:
```suggestion
In this model, `-1` denotes that there are no landmarks. Using this landmark information, you can create a mapping file. Suppose you create a mapping file with the name `pose_mapping.txt`, then you can populate the `pose_mapping.txt` file as follows:
```

Reply:
@tyoungroy 
I think you have missed this comment.